integration/tests/basic/test_payment_in_different_tokens.py

```diff
@@ -0,0 +1,141 @@
+import allure
```

Comment:
for testing only SOL network I think we could add:
1. check that SOL balance changed for user (to verify that network uses spl token for all payments)
2. Operator got fee in Sol?
3. Check that Sol for user changed in NEON network
4. Check that user in neon network could send Sol and Neon in Sol network
5. In issue I wrote a several cases with contracts for inter-network interaction